src/heuristicblocking.js

```diff
@@ -181,9 +204,6 @@ HeuristicBlocker.prototype = {
     if(firstParties.indexOf(page_origin) === -1){
       firstParties.push(page_origin);
       snitch_map.setItem(tracker_origin, firstParties);
-      // TODO removing this line/messing up parameters should break integration tests
-      this.storage.setupHeuristicAction(tracker_fqdn, constants.ALLOW);
-      this.storage.setupHeuristicAction(tracker_origin, constants.ALLOW);
```

Comment:
As for tests, QUnit is failing because `action_map.getItem('testsite.com').heuristicAction === "allow"` is no longer truthy, which makes sense.